waspc/cli/src/Wasp/Cli/Command/BuildStart/Server.hs

```diff
@@ -0,0 +1,75 @@
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+{-# HLINT ignore "Use fewer imports" #-}
+module Wasp.Cli.Command.BuildStart.Server
+  ( buildServer,
+    startServer,
+  )
+where
+
+import Data.Function ((&))
+import StrongPath ((</>))
+import qualified StrongPath as SP
+import System.Process (proc)
+import Wasp.Cli.Command.BuildStart.Config (BuildStartConfig)
+import qualified Wasp.Cli.Command.BuildStart.Config as Config
+import qualified Wasp.Job as J
+import Wasp.Job.Except (ExceptJob, toExceptJob)
+import Wasp.Job.Process (runProcessAsJob)
+import Wasp.Project.Env (dotEnvServer)
+import Wasp.Util.Random (genRandomAsciiAlphaNumsIO)
+
+buildServer :: BuildStartConfig -> ExceptJob
+buildServer config =
+  runProcessAsJob
+    (proc "docker" ["build", "--tag", dockerImageName, contextPath])
+    J.Server
+    & toExceptJob (("Building the server failed with exit code: " <>) . show)
+  where
+    buildDir = Config.buildDir config
+    contextPath = SP.fromAbsDir buildDir -- The folder with the Dockerfile and the files it needs to build.
```

Comment:
```suggestion
    dockerContextDir = SP.fromAbsDir buildDir
```

I removed `Path` and put `Dir` because that is more consistent with how we name other stuff (e.g. `buildDir`).
Also, `docker` in front helps.
Finally, I don't thikn comment is needed, because it is now easy to google `docker context`.